mapper/dataset_test.go

```diff
@@ -244,9 +248,20 @@ func TestCreateDatasetPage(t *testing.T) {
 		serviceMessage := getTestServiceMessage()
 		emergencyBanner := getTestEmergencyBanner()
 
+		// get cached navigation data
+		cfg, err := config.Get()
```

Comment:
The `cfg` should generally be initialised in the `Given` level of the test or above that